src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/basicshortmutpileup/BetaBinomialDistributionUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.tools.walkers.validation.basicshortmutpileup;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.junit.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class BetaBinomialDistributionUnitTest extends BaseTest {
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`

Reply:
Done